### DISCUSSION TITLE
7 failed updated while reinstalling moose

### DISCUSSION THREAD

**Discussion by Spudman105** [idaholab/moose#26961] [Q&A Getting Started] [answered]
I am reinstalling moose and the first time had no problems but this time I am having the same 7 errors show when i run the main test on moose.

```pre
utils/param_error.error ...................................................... FAILED (EXPECTED ERROR MISSING)
parser/include.include ........................................................................ FAILED (CRASH)
parser/multiple_inputs.merging/three_inputs_error ............................ FAILED (EXPECTED ERROR MISSING)
parser/multiple_inputs.merging/three_inputs_override_message ................ FAILED (EXPECTED OUTPUT MISSING)
misc/hit_cli.merge ........................................................................... FAILED (CODE 1)
misc/hit_cli.diff/vanilla .................................................................... FAILED (CODE 1)
misc/hit_cli.diff/color ...................................................................... FAILED (CODE 1)
--------------------------------------------------------------------------------------------------------------
Ran 4228 tests in 199.2 seconds. Average test time 0.3 seconds, maximum test time 12.2 seconds.
4221 passed, 96 skipped, 0 pending, 7 FAILED
```

**Comment by GiudGiud** [comment]
Hello

We need the actual log, not just the summary at the end, to diagnose what is happening
If you run `./run_tests --failed-tests` you ll see it

Guillaume

**Reply by Spudman105** [reply]
Its fixed but in the future I will try to give more information! thanks for the advice!

**Comment by brandonlangley** [comment, marked as answer]
@Spudman105 -

Those specific tests failing probably indicate your WASP version is out of date with your MOOSE version.

If you are using conda packages for these components, you can update to the latest version by running:

```
conda update moose-dev
```

And if that does not work, then you can try to remove and reinstall a new moose conda environment with:

```
conda env remove -n moose -all
conda create -n moose moose-dev
```

If you are not using conda and are manually installing everything, you can run this from the MOOSE root:

```
./scripts/update_and_rebuild_wasp.sh
```

I would suggest to then clean up your repo with `git clean -xfd`, rebuild MOOSE, and rerun those tests.

Note that `git clean -xfd` will delete any uncommitted work, so make sure you save your changes first.

**Reply by Spudman105** [reply]
I had updated everything earlier but the clean helped and I'm not getting any failed tests. Thank you!
